markdown/en/api/ff-searchbox.api.md

```diff
@@ -41,7 +41,7 @@ ___
 | **add-params**&nbsp;(String) (default: empty)| _The parameter string has to be URL-encoded._ With this property you can deliver standard parameters which will then be attached to the search request. Example: add-params="param1=abcd,param2=xyz"|
 | **add-tracking-params**&nbsp;(String) (default: empty)| With this property you can deliver standard parameters which are attached to every tracking request. Example: add-tracking-params="param1=abcd,param2=xyz"|
 | **keep-filters**&nbsp;(Boolean) **Options**: &nbsp;true, &nbsp;false (default: false)| With this property you can determine, if filters which where set before the search (e.g. via ASN) should be kept or discarded.|
-| **keep-url-params**&nbsp;(String) (default: empty)| Comma separated values; specifies the url-parameters that should be included in requests to FACT-Finder. If this is set to `all`, every parameter will be used |
+| **keep-url-params**&nbsp;(String) (default: empty)| Comma separated values; specifies the url-parameters that should be included in requests to FACT-Finder. *This is particularly useful if you only want to keep your own custom non-FF attributes in the URL.* If this is set to `all`, every parameter will be used |
```

Comment:
Really? If you "**only** want to keep your own"? Sounds like it would throw away other parameters. Does it?

Would you also consider placing it at the end of the functional explanation? So it becomes like a side note.